### PR TITLE
feat(TG-434): Refactoring dashboard / map / views to save correct structure into DB

### PR DESCRIPTION
**Pull request by Appius**
Refactored dashboard, mapView and view structures to correctly match requirements.
Implemented view save.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/99" target="_blank">ReviewNinja

File: src/API/dashboard-view/dashboard-view.service.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import DashboardViewModel from './dashboard-view.model';
+import        SensorModel from '../sensors/sensor.model';
```
Comment (borys-ihnatyev) [inline src/API/dashboard-view/dashboard-view.service.js]: we did not have such align in code style

File: src/API/map-view/map-view.model.js [hunk 2]
```diff
@@ -31,25 +30,10 @@ export default moongose.model('MapView', new moongose.Schema({
         default: '',
         required: false
```
Comment (borys-ihnatyev) [inline src/API/map-view/map-view.model.js]: I would make it schema rather than model

File: src/API/dashboard-view/dashboard-view.service.js (hunk 1 above)
Comment (Appius) [inline src/API/dashboard-view/dashboard-view.service.js]: !fixed

**Pull request merged**